docs/www/docs/contribute/99-troubleshooting.md

```diff
@@ -66,31 +66,31 @@ It can happen that the host firewall can block downloading these binaries. If us
 ```json
 {
   "version": "stable",
-  "buildId": "1.0.020761",
-  "publishDate": "2022-09-07T17:34:35.4261936Z",
+  "buildId": "<Version of the StaticSiteClient>",
+  "publishDate": "<publishDate>",
   "files": {
     "linux-x64": {
-      "url": "https://swalocaldeploy.azureedge.net/downloads/1.0.020761/linux/StaticSitesClient",
-      "sha": "871ae9f154ddb8123e24c507e39dfc11c99f34815e59b178051a6e44a870e308"
+      "url": "https://swalocaldeploy.azureedge.net/downloads/<Version of the StaticSiteClient>/linux/StaticSitesClient",
+      "sha": "<Hash value of the Linux Binary>"
```

Comment:
<Hash value of StaticSitesClient Linux binary> 

Similarly check at other places